### PR TITLE
chore: improve .gitignore and remove .DS_Store

### PR DESCRIPTION
**Pull request by Anmol-Baranwal**
### Related Issue 

Closes: #966

#### Changes made 👷🏻‍♂️

- Removed `.DS_Store`
- Updated `.gitignore` (I have avoided duplicate ones)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,9 +1,12 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 node_modules
+
 # dependencies
 /node_modules
 /.pnp
 .pnp.js
+./package-lock.json
+package-lock.json
```
Comment (tamalCodes) [inline .gitignore:9]: Please remove `./package-lock.json` and `package-lock.json` from here. They are files which needs to be in our repo for builds to kick-in.

File: .gitignore [hunk 2]
```diff
@@ -14,13 +17,159 @@ node_modules
...
+.env
 .env.local
 .env.development.local
 .env.test.local
 .env.production.local
 
+# next.js
+/.next/
+/out/
+
```
Comment (tamalCodes) [inline .gitignore:33]: Please remove files related to nextjs, we are not using these files anyways.

File: .gitignore [hunk 3]
```diff
@@ -14,13 +17,159 @@ node_modules
...
+# vercel
 .vercel
+
+# typescript cache
+*.tsbuildinfo
+
+# typeScript declaration
+next-env.d.ts
+
```
Comment (tamalCodes) [inline .gitignore:51]: There is no need to add anything related to Typescript as of now. Please remove these.

File: .gitignore [hunk 4]
```diff
@@ -14,13 +17,159 @@ node_modules
...
+# Microbundle cache
+.rpt2_cache/
+.rts2_cache_cjs/
+.rts2_cache_es/
+.rts2_cache_umd/
+
+# Optional REPL history
+.node_repl_history
+
```
Comment (tamalCodes) [inline .gitignore:107]: Please remove `REPL history` from here, we are anyways not using them.

File: .gitignore [hunk 5]
```diff
@@ -14,13 +17,159 @@ node_modules
...
+# Output of 'npm pack'
+*.tgz
+
+# Yarn Integrity file
+.yarn-integrity
+
```
Comment (tamalCodes) [inline .gitignore:113]: No need to add `yarn` files either, we are anyways not using them.

File: .gitignore [hunk 6]
```diff
@@ -14,13 +17,159 @@ node_modules
...
+
+# dotenv environment variable files
 .env
+.env.development.local
+.env.test.local
+.env.production.local
+.env.local
+
```
Comment (tamalCodes) [inline .gitignore:121]: Please remove everything related to env here, we have those in line number 26.

**Review by tamalCodes — CHANGES_REQUESTED** [review]
Please remove all declarations of files from `.gitignore` whose chances of getting used are none. Like yarn, next, grunt, istanbul and so on. We need not have our `.gitignore` so long and ignore all possible things. 

Please update accordingly.

**Comment by Anmol-Baranwal** [comment]
@tamalCodes 

I can still do this till 16th since I am very busy with Web3Conf, so I am not able to devote proper time.

**Comment by tamalCodes** [comment]
So glad to hear that, I closed majority Issues because most of them were targeted at GSSOC with not much actual intrest in OSS. I'll re-open this up.

**Comment by Anmol-Baranwal** [comment]
Done.

**Comment by Anmol-Baranwal** [comment]
@tamalCodes 
Feel free to tag me, if there are any open issues, that you want me to take.

**Pull request merged**